src/elements/tokens/TokenTable.tsx

```diff
@@ -11,9 +11,16 @@ import { ReactComponent as IconSearch } from 'assets/icons/search.svg';
 import { NavLink } from 'react-router-dom';
 import { wethToken } from 'services/web3/config';
 
-export const TokenTable = () => {
+
+interface Props {
+  searchInput: string;
+  setSearchInput: Function;
+} 
+
+
+export const TokenTable = ({ searchInput,setSearchInput }: Props) => {
```

Comment:
Please use Prettier to enforce code styling

Reply:
done